sources/packages/api/src/services/student-account-applications/student-account-applications.service.ts

```diff
@@ -134,6 +134,11 @@ export class StudentAccountApplicationsService extends RecordDataModelService<St
       city: studentProfile.city,
       postalCode: studentProfile.postalCode,
       selectedCountry: studentProfile.selectedCountry,
+      // The Ministry approval will happen based on a student profile submitted form
+      // that contains the SIN consent. If the consent is not provided by the student
+      // the Ministry approval would never happen, hence we are considering at this
+      // moment that the student provided already the SIN consent.
+      sinConsent: studentProfile.sinConsent,
```

Comment:
As we talked, this value should come from the DB probably with the query already executed on line 105.

Reply:
Updated as discussed.